kumascript/macros/WebRTCSidebar.ejs

```diff
@@ -31,6 +31,27 @@ var text = mdn.localStringMap({
     'The_MDN_project': 'The MDN project',
     'build_a_phone_with_peer_js': 'Building an internet-connected phone with Peer.js',
   },
+  'fr': {
+    'WebRTC_API': "API WebRTC",
+    'WebRTC_guides': "Guides WebRTC",
+    'Architecture': "Architecture WebRTC",
+    'Basics': "Fondamentaux WebRTC",
+    'Protocols': "Protocoles WebRTC",
+    'Connectivity': "Gérer la connectivité",
+    'Overview': "Aperçu des interfaces WebRTC",
+    'Session_lifetime': "Cycle de vie d'une session WebRTC",
+    'Using_data_channels': "Utiliser les canaux de données",
+    'WebRTC_tutorials': "Tutoriels WebRTC",
+    'adapter_js': "Interopérabilité avec adapter.js",
+    'still_photos': "Prendre des photos avec la camera",
+    'simple_sample': "Un exemple simple de canal de données",
+    'Interfaces': "Interfaces",
+    'Documentation': "Documentation:",
```

Comment:
I missed this earlier, the suggestion is to use `\xa0` because `&nbsp;` doesn't seem to work (cf. https://github.com/mdn/yari/pull/6218#discussion_r869852622)

```suggestion
    'Documentation': "Documentation\xa0:",
```